code/game/atoms_movable.dm

```diff
@@ -535,6 +536,37 @@
 			return
 	A.Bumped(src)
 
+/atom/movable/Exited(atom/movable/AM, atom/newLoc)
+	. = ..()
+	if(AM.area_sensitive_contents)
+		for(var/k in get_nested_locs(src) + src)
+			var/atom/movable/location = k
+			LAZYREMOVE(location.area_sensitive_contents, AM.area_sensitive_contents)
+
+/atom/movable/Entered(atom/movable/AM, atom/oldLoc)
+	. = ..()
+	if(AM.area_sensitive_contents)
+		for(var/k in get_nested_locs(src) + src)
+			var/atom/movable/location = k
+			LAZYADD(location.area_sensitive_contents, AM.area_sensitive_contents)
+
+//See traits.dm. Use this in place of ADD_TRAIT.
```

Comment:
```suggestion
/// See traits.dm. Use this in place of ADD_TRAIT.
```

Reply:
The important part is the `///`, I just added an extra space because that's what I like 🙂